apps/web/src/hooks/v3/useV3DerivedInfo.ts

```diff
@@ -185,6 +187,8 @@ export default function useV3DerivedInfo(
           : (!invertPrice && typeof rightRangeTypedValue === 'boolean') ||
             (invertPrice && typeof leftRangeTypedValue === 'boolean')
           ? tickSpaceLimits[Bound.UPPER]
+            ? tickSpaceLimits[Bound.UPPER] - 1
```

Comment:
We need to consider the tick spacing. For example if tick spacing is 10, then the tickSpaceLimit here won't be 887272, but 887270. In this case there's no need to minus 1

Reply:
Done